File: Array/Python/two-sum.py
nums = [3,2,4]
target = 6
result = []

for i in range(len(nums)):
    for j in range(i+1, len(nums)):
        if nums[i] + nums[j] == target:
            result.append(i)
            result.append(j)
print(result)

File: Array/C++/two-sum.cpp
#include <bits/stdc++.h>
#define log(x) cout<<x<<endl;
using namespace std;

void print(vector<int> a){
    for(auto i: a){
        cout<<i<<" ";
    }
    cout<<endl;
}

int main()
{
    vector<int> nums= {3,2,4}; int target = 6;
    vector<int> result;
    for(int i=0; i<nums.size(); i++){
        for(int j=i+1; j<nums.size(); j++){
            if(nums[i] + nums[j] == target) {
                result.push_back(i);
                result.push_back(j);
                break;
            }
        }
    }
    print(result);  
}
